src/Cake.Issues.EsLint.Tests/Cake.Issues.EsLint.Tests.csproj

```diff
@@ -29,17 +29,18 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
     <PackageReference Include="Cake.Core" Version="0.22.0" />
     <PackageReference Include="Cake.Testing" Version="0.22.0" />
-    <PackageReference Include="Cake.Issues" Version="0.2.0-beta0001" />
-    <PackageReference Include="Cake.Issues.Testing" Version="0.2.0-beta0001" />
+    <PackageReference Include="Cake.Issues" Version="0.2.0-beta0002" />
+    <PackageReference Include="Cake.Issues.Testing" Version="0.2.0-beta0002" />
     <PackageReference Include="Shouldly" Version="2.8.3" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.0.2" />
-    <PackageReference Include="xunit" Version="2.2.0" />
+    <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.abstractions" Version="2.0.1" />
-    <PackageReference Include="xunit.assert" Version="2.2.0" />
-    <PackageReference Include="xunit.core" Version="2.2.0" />
-    <PackageReference Include="xunit.extensibility.core" Version="2.2.0" />
-    <PackageReference Include="xunit.extensibility.execution" Version="2.2.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
+    <PackageReference Include="xunit.analyzers" Version="0.8.0" />
```

Comment:
Why this package?

Reply:
`xunit.analyzers` contains rules for checking best practices for writing XUnit tests. It is a dependency (in version 0.7.0) on the XUnit package